autogpt/agent/agent.py

```diff
@@ -163,6 +163,14 @@ def signal_handler(signum, frame):
                     print_assistant_thoughts(
                         self.ai_name, reply_content_json, self.config.speak_mode
                     )
+                    self.log_cycle_handler.log_cycle(
+                        self.ai_config.ai_name,
+                        self.created_at,
+                        self.cycle_count,
+                        reply_content_json,
+                        NEXT_ACTION_FILE_NAME,
+                    )
```

Comment:
I still don't like it, because I want to log content and functions, but at least it's better